src/components/layout/SpinnerButton/SpinnerButton.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import { ReactComponent as SpinnerIcon } from 'components/layout/SpinnerButton/assets/loader.svg'
+
+const SpinnerButton = ({ className, disabled, isLoading, onClick, value }) => {
+  return (
+    <button
+      className={'primary-button loading-spinner ' + className}
+      disabled={disabled || isLoading}
+      onClick={onClick}
+      type="submit"
+    >
+      {isLoading ? <SpinnerIcon /> : value}
```

Comment:
```suggestion
      {isLoading ? <SpinnerIcon /> : children}
```
Ce qui implique de changer l'appel de ce composant : `<SubmitButton {...props}>Enregistrer</SubmitButton>`.
Potentiellement, on pourrait même ajouter du html.